Resources/config/controller.xml

```diff
@@ -17,6 +17,7 @@
             <argument type="service" id="liip_monitor.helper.runner_manager" />
             <argument type="service" id="liip_monitor.helper" />
             <argument>%liip_monitor.view_template%</argument>
+            <argument type="service" id="logger" on-invalid="ignore" />
```

Comment:
I fear it means we then require that the MonologBundle is active .. 

@stof can you confirm this? what is the current best practice here? I seem to remember that we are phasing out ``if (null !== $this->logger) {`` type code in core .. and afaik also making ``NullLogger`` fallback instances ..